static/docs/command-reference/repro.md

```diff
@@ -37,7 +37,8 @@ omitted, `Dvcfile` will be assumed.
 
 By default, this command recursively searches in pipeline stages, starting from
 the `targets`, to determine which ones have changed. Then it executes the
-corresponding commands.
+corresponding commands.<br /> Note that DVC removes cached <abbr>outputs</abbr>
```

Comment:
hmm, why do we need html here?

Reply:
New line between the first sentence and the note:

![image](https://user-images.githubusercontent.com/1477535/71283474-917af680-231d-11ea-99e6-a814c48a15eb.png)

Idk. Stylistic decision. There's a few `<br />`s here and there in docs. Should I review and try to remove them?

Reply:
👍 I think it's fine